docs/introduction/flows.md

```diff
@@ -12,15 +12,44 @@ Intent describes some potential future event which might require prior Commitmen
 
 ## Commitment
 
-Commitments are "mirrors" of economic events, and economic events can fulfull commitments. Commitments can be considered contractual promises from one agent to another.
+Commitments are "mirrors" of economic events, and economic events can fulfull commitments.  Commitments can satisfy intents. Commitments can be considered contractual promises from one agent to another.
 Commitments describe potential future event which involved agents already agreed to pursuit.
 
 ## Economic Events
 
-Economic event describes past event, something already observed, never some potential future event.
+Economic event describes past event, something already observed, never some potential future event.  They can fulfill commitments or intents.
 
 ## Granularity
 
 Intents, Commitments, and Economic Events can occur at any granularity that is needed or for which data can be obtained.  So they primarily are used for all operational needs, but can also be used at higher levels for analytical and high level planning needs for communities or regions, for example.
 
 ![Intent-Commitment-Event](https://rawgit.com/valueflows/valueflows/master/release-doc-in-process/i-c-e.png)
+
+
+## Actions
+
+All flows use an action property to designate what the flow is doing and how it will affect an economic resource (or not).  Actions have defined behaviors as follows.
+
+Action | Affect | Related to | I/O | Pairs with |
+------ | ------ | ---------- | --- | ---------- |
+increment | Increment | N/A | N/A | N/A |
+produce | Increment | Process | Output | N/A |
+improve | No effect | Process | Output | accept |
+unload | Increment | Process | Output | load |
+receive | Increment | Transfer | Output | give |
+decrement | Decrement | N/A | N/A | N/A |
+load | Decrement | Process | Input | unload |
+consume | Decrement | Process | Input | N/A |
+use | No effect | Process | Input | N/A |
+work | No effect | Process | Input | N/A |
+cite | No effect | Process | Input | N/A |
+accept | No effect | Process | Input | improve |
+give | Decrement | Transfer | Input | receive |
+
+We have defined a core set of actions, but expect that this will be extended with others. If extended, they should be defined as part of this or another formal vocabulary so that all can use them and assume the same meaning. 
+
+Note: Increment and decrement are usually not meant to be used directly, rather the actions inherit from them.  An exception is if an adjustment is needed that has no known provenance.  Examples of when this might happen: 
```

Comment:
I disagree with *Increment and decrement are usually not meant to be used directly*. We should clarify in detail what each of the action implies (how applications should interpret and reason over them) and just recommend to use action which has the most accurate implications.

Reply:
#355 (hard vs. soft allocations) may need simple decrement on one resource and increment on another one, but since it does not change existence probably one can just use *give* (former *issue*) and *receive* without implying any changes to 'rights and responsibilities'

Reply:
removed that sentence; removed from chart

Reply:
I found a couple verbs for adjust: raise, lower.  So I don't think we need or want increment and decrement as actual actions, just abstract inheritance for behavior.  Make sense to you?  People should use the more defined actions imo, and define others as needed.